projects/gearbox/index.js

```diff
@@ -406,13 +409,17 @@ async function getV3CAs(dc300, creditManager, block, api) {
 }
 
 // src/adapter/index.ts
-async function tvl(api) {
-  const block = api.block
+async function tvl(timestamp, block, _, { api }) {
```

Comment:
this is mostly good except for this line, the block there is always ethereum block (it is legacy)

so, what you want is `cont block = await api.getBlock()`